src/main/java/core/basesyntax/model/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.model;
+
+public class Circle extends Figure implements Draw, GetArea {
```

Comment:
```suggestion
public class Circle extends Figure {
```

если клас Figure имплементит Draw и GetArea, то все его наследники автоматом имплиментят  Draw и GetArea